rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -818,11 +818,7 @@ def get_multisite_info():
         zone_names = zone_names + zone_name
         if i != len(period_list.get("period_map")["zonegroups"][0]["zones"]) - 1:
             zone_names = zone_names + ","
-    op = utils.exec_shell_cmd("radosgw-admin sync status")
-    lines = list(op.split("\n"))
-    for line in lines:
-        if "realm" in line:
-            realm_name = line[line.find("(") + 1 : line.find(")")]
+    realm_name = period_list.get("realm_name")
```

Comment:
@viduship
I think you are reverting changes merged in this PR by @anrao19
https://github.com/red-hat-storage/ceph-qe-scripts/pull/556/files

could you please check..